src/lib.rs

```diff
@@ -996,6 +996,55 @@ impl Build {
             }
         }
 
+        #[cfg(feature = "cudart")]
```

Comment:
I think it might be best to use `if self.cuda && cfg!(feature = "cudart") { .. }` here since that means we can type-check this for other platforms instead of having cuda-only issues cropping up in the future.

Reply:
(same for some of the checks below)

Reply:
> (same for some of the checks below)

You mean ones for linux and msvc?

Meanwhile for reference, the original reason for `#[cfg(feature = "cudart")]` was to make dependency on `which` optional. I mean majority don't use cuda, and so they wouldn't need `which`...

Reply:
Ah good point about the conditional `which` dependency, but in any case the current structure of the code is what I was thinking of, thanks!